components/features.tsx

```diff
@@ -13,6 +16,19 @@ type SingleFeatureProps = {
 };
 
 export default function Features() {
+  const [keybind, setKeybind] = useState("");
+
+  useEffect(() => {
+    const platform = navigator.platform.toLowerCase();
+    if (platform.includes("mac")) {
+      setKeybind("CMD+I");
+    } else if (platform.includes("win")) {
+      setKeybind("CTRL+I");
+    } else {
+      setKeybind("CTRL+I");
+    }
+  }, []);
+
```

Comment:
hmmm i think there is a better way to do this since navigator.platform is deprecated and dosent support all browsers. I think u can use next/headers or something similar. But since that would be a server component you will have to split the client and server component. 
```typescript
import { headers } from 'next/headers'
 const headersList = headers();
  const userAgent = headersList.get('user-agent') || '';

  if (userAgent.toLowerCase().includes('mac')) {
    return 'CMD+I';
  }
  return 'CTRL+I';
}
```